api/app/tests/wildfire_one/test_wildfire_one.py

```diff
@@ -110,3 +114,19 @@ async def run_test():
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
     loop.run_until_complete(run_test())
+
+
+class PostResponse():
+    def __init__(self, status):
+        self.status = status
+
+    async def json(self):
+        return {}
+
+
+@pytest.mark.anyio
+@patch('app.wildfire_one.wfwx_post_api.ClientSession')
+async def test_wf1_post_failure(mock_client):
+    mock_client.post.return_value.__aenter__.return_value = PostResponse(status=401)
+    with pytest.raises(ClientConnectionError):
+        await post_forecasts(mock_client, 'token', [])
```

Comment:
Got the test working. If you have suggestions or improvements for this, let me know

Reply:
This is good!